src/periodograms.jl

```diff
@@ -601,24 +607,26 @@ See [`periodogram`](@ref) for description of optional keyword arguments.
 ```jldoctest
 julia> x = rect(10; padding=20);
 
-julia> power(welch_pgram(x))                    # 1-sided periodogram
+julia> power(welch_pgram(x; window=nothing))                    # 1-sided periodogram
 2-element Vector{Float64}:
  0.9523809523809523
  0.04761904761904761
 
-julia> power(welch_pgram(x; onesided=false))    # 2-sided periodogram
+julia> power(welch_pgram(x; onesided=false, window=nothing))    # 2-sided periodogram
 3-element Vector{Float64}:
  0.9523809523809523
  0.023809523809523805
  0.023809523809523805
 
```

Comment:
Due to the short length of `x`, the default `n==3` and
```julia
julia> hanning(3)
3-element Vector{Float64}:
 0.0
 1.0
 0.0
```
is not at all better than the default rect window. In the next example, where `n=5` is explicitly set, I've changed the window to `hanning` as it somewhat reasonable there. I think from a pedagogical viewpoint this is ok, and I'd like to keep a complete rewrite of the example out of the scope of this PR.

Sidenote: a variant of the windows without the zeros at the edges might a useful here, e.g.
```julia
julia> hanning(5)[begin+1:end-1]
3-element Vector{Float64}:
 0.5
 1.0
 0.5
```
But that's also beyond the scope of this PR.